ROADMAP.md

```diff
@@ -6,96 +6,49 @@ Community and contributor involvement is vital for successfully implementing all
 
 OpenEBS follows a lean project management approach by splitting the development items into current, near term and future categories.
 
+## Near-term and long-term roadmap (2024 onwards)
+This table contains a list of both near-term and long-term backlog items. Near-term features are currently getting the most active focus and attention within the project. These are prioritized and planned to be completed within the next major release where higher priority is given to usability, stability, resilience, data integrity issues reported by the community. The long-term features require significant design and development efforts, and will be scoped in the future.
+
+OpenEBS follows a release cadence with a new minor release every 3-4 months.
+
+_Note_: The planned release timelines, version numbers and feature priorities are subject to change as the project maintainers/leadership/community continuously update and adjust in response to K8s industry movements, trends and our community influence.
+
+_Note 2_: The table contains the high-level backlog items. For a comprehensive list of features and issues, please refer to https://github.com/openebs/openebs/releases.
+
+| Feature | Description | Local or Replicated | Release timeline | Status |
+| :------ | :---------- | :------------------ | :--------------- | :----- |
+| Unified installer | Unified Helm installer for all engines, deprecates operator yaml | All | v4.0 (Q1 2024) | Completed |
+| Unified documentation | Unified and restructured documentation website, deprecates mayastor.gitbook.io | All | v4.0 (Q1 2024) | Completed |
+| Legacy engines deprecation | Deprecated, archived and removed support for legacy engines and components eg. CStor, Jiva, NFS, NDM | All | v4.0 (Q1 2024) | Completed |
+| Volume resize | Able to increase volume size and overlaying filesystem size with I/O continuity | Replicated PV Mayastor | v4.0 (Q1 2024) | Completed |
+| Multi-replica volume snapshot and cloning | Able to take consistent snapshots across all available replicas of a volume and restore to a given snapshot | Replicated PV Mayastor | v4.1 (Q3 2024) | Completed |
+| Local PV CI | CI hardening and enhancements, helm chart support and more tests | Local PV (LVM, ZFS, Hostpath) | v4.2 (Q4 2024) | In progress |
+| Local PV E2E | E2E hardening, umbrella chart testing, conversion of Ansible to Ginkgo-based BDDs | Local PV (LVM, ZFS, Hostpath) | v4.2 (Q4 2024) | In progress |
+| Replica topology | Replica distribution based on pool and node topologies | Replicated PV Mayastor | v4.2 (Q4 2024) | In progress |
+| NVMe-oF over RDMA support | Support for NVMe-oF over RDMA as transport | Replicated PV Mayastor | v4.2 (Q4 2024) | In progress |
+| Unified kubectl plugin | Unified kubectl plugin to manage all OpenEBS components | All | v4.2 (Q4 2024) | In progress |
+| Local PV LVM cloning | Able to do K8s restore of Local PV LVM snapshot | Local PV LVM | v4.3 (Q1 2025) | In progress |
```

Comment:
Some upcoming items are marked `In-progress` and some have status blank. `In-progress` doesn't seem right status for these.

Reply:
The upcoming items marked as in-progress already have ongoing efforts. The blank ones haven't been started. Hope that explains.

Reply:
Fixed the status on Local PV LVM cloning feature.